CODEOWNERS

```diff
@@ -1,7 +1,3 @@
-# Each line is a file pattern followed by one or more owners.
+# These owners will be the default owners for everything in the repo.
 
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence,
-# these owners will be requested for
-# review when someone opens a pull request.
-*       @steffenkleinle @ztefanie @f1sh1918 @LeandraH
+-       @steffenkleinle @f1sh1918 @LeandraH
```

Comment:
Perhaps we should add @bahaaTuffaha? :)

Reply:
What do you think @bahaaTuffaha ? Do you feel ready to be a code owner?

Reply:
Hmm if you feel this is best for the team I don't have a problem...
And sad to see Steffi leaving 😢

Reply:
Nothing to worry about, basically the only thing changing with this is that you get automatically get added as reviewer for all PRs in the repo and that your approvals/requested changes allow/disallow for merging :)